Align team

emctackett opened the pull request:
This PR added the appropriate links for each team member's email, GitHub, and LinkedIn to the `href` associated with the mail/GH/LI icons.

I also attempted to align photos by adjusting their size... it looked like this before:

<img width="723" alt="o Lodoux" src="https://user-images.githubusercontent.com/19399698/130087783-19174903-3fb9-4bdb-ac06-6f1349cbda74.png">

Currently the size of the images isn't responsive to screen size so we might want to add that?

emctackett committed:
assign consistent height and width to team photos so they remain aligned

emctackett committed:
add email, LI, and GH links for each team member

emctackett committed:
try a slightly resized photo for jimmy

kaledoux reviewed (changes requested):
The images are all the same resolution now, but Jimmy's is stretched and the auto-centering formatting is a bit off. The images and the text/links below them aren't aligned properly.
![Screenshot from 2021-08-19 11-09-43](https://user-images.githubusercontent.com/42515081/130093742-f7113098-64c6-4e04-a134-f0131a5bee0a.png)
I think we'll need an updated photo from Jimmy that has a higher resolution to fix the stretching. The misalignment is likely just a bit of CSS tweaking.

emctackett committed:
add jekyll config and README for local development

emctackett committed:
add jekyll config and readme for local development. update CSS for photo alignment

emctackett commented:
The alignment of the photos should be fixed now; I added an additional class to the css. 

I was having trouble running the different branches locally and added a jekyll config. You should be able to run any branch fully by running `jekyll build` and then `jekyll serve` on that branch. The local version will run on port :`4000`.
